From: 13977028965 <1592941704@qq.com>
Date: Sun, 8 Jun 2025 15:38:50 +0000
Subject: Translated using Weblate (Vietnamese)

Currently translated at 100.0% (12 of 12 strings)

Translation: openkylin-nile-new/vpn
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/vpn/vi/
---
 plugins/vpn/translations/vi.ts | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/plugins/vpn/translations/vi.ts b/plugins/vpn/translations/vi.ts
index 661944d..e336cd2 100644
--- a/plugins/vpn/translations/vi.ts
+++ b/plugins/vpn/translations/vi.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../../component/AddBtn/addnetbtn.cpp" line="44"/>
         <source>Add Others</source>
-        <translation>Thêm tùy chọn khác</translation>
+        <translation>Tham gia vào mạng khác</translation>
     </message>
     <message>
         <location filename="../../component/AddBtn/addnetbtn.cpp" line="48"/>
@@ -19,7 +19,7 @@
     <message>
         <location filename="../itemframe.cpp" line="38"/>
         <source>Add VPN</source>
-        <translation type="unfinished"></translation>
+        <translation>Thêm VPN</translation>
     </message>
 </context>
 <context>
@@ -39,26 +39,26 @@
         <location filename="../vpn.cpp" line="157"/>
         <location filename="../vpn.cpp" line="284"/>
         <source>Show on Taskbar</source>
-        <translation type="unfinished"></translation>
+        <translation>Hiển thị biểu tượng trên thanh tác vụ</translation>
         <extra-contents_path>/Vpn/Show on Taskbar</extra-contents_path>
     </message>
     <message>
         <location filename="../vpn.cpp" line="286"/>
         <source>Add VPN</source>
-        <translation type="unfinished"></translation>
+        <translation>Thêm kết nối VPN</translation>
         <extra-contents_path>/Vpn/Add VPN</extra-contents_path>
     </message>
     <message>
         <location filename="../vpn.cpp" line="372"/>
         <location filename="../vpn.cpp" line="479"/>
         <source>connected</source>
-        <translation type="unfinished"></translation>
+        <translation>Đã kết nối</translation>
     </message>
     <message>
         <location filename="../vpn.cpp" line="374"/>
         <location filename="../vpn.cpp" line="489"/>
         <source>not connected</source>
-        <translation type="unfinished"></translation>
+        <translation>Chưa kết nối</translation>
     </message>
 </context>
 <context>
@@ -66,19 +66,19 @@
     <message>
         <location filename="../vpnitem.cpp" line="55"/>
         <source>Delete</source>
-        <translation type="unfinished"></translation>
+        <translation>Xóa</translation>
     </message>
     <message>
         <location filename="../vpnitem.cpp" line="107"/>
         <location filename="../vpnitem.cpp" line="120"/>
         <source>Disconnect</source>
-        <translation type="unfinished"></translation>
+        <translation>Ngắt kết nối</translation>
     </message>
     <message>
         <location filename="../vpnitem.cpp" line="109"/>
         <location filename="../vpnitem.cpp" line="118"/>
         <source>Connect</source>
-        <translation type="unfinished"></translation>
+        <translation>Kết nối</translation>
     </message>
 </context>
 </TS>
